src/stylus/fablevision/target-drake.styl

```diff
@@ -1,14 +1,16 @@
 .geniblocks.target-drake-container
   display: flex
   flex-direction: row
-  height: 204px
-  width: 520px
+  height: 195px
+  width: 500px
   background: url('../resources/fablevision/layout-c/target_drake_gizmo.png')
   background-repeat: no-repeat
   background-size: cover
   justify-content: center
   align-self: center
   z-index: 3
+  position:absolute
+  bottom:0px
```

Comment:
I played around with this in the debugger and with a value of `2px` the gizmo appears to rest on the bottom HUD rather than overlapping it slightly on both Chrome and Firefox. Other than that, this is much simpler than the code it replaces.